DependencyInjection/TrsteelCkeditorExtension.php

```diff
@@ -43,39 +43,9 @@ public function load(array $configs, ContainerBuilder $container)
             'Cache.SerializerPath' => '%kernel.cache_dir%',
         ), $config['html_purifier']['config']);
 
+        $container->setParameter('trsteel_ckeditor.ckeditor', $config);
```

Comment:
I'm just talking about the variables you are giving to the form type. e.g. you don't need to give it the class etc?

Also, changing the constructor signature is going to be a BC anyway incase anyone is extending the class itself?